pycbc/events/coinc.py

```diff
@@ -351,17 +351,17 @@ def cluster_coincs(stat, time1, time2, timeslide_id, slide, window, argmax=numpy
     Returns
     -------
     cindex: numpy.ndarray
-        The set of indices corresponding to the surviving coincidences.
+        The set of indices corrsponding to the surviving coincidences
     """
+
     logging.info('clustering coinc triggers over %ss window' % window)
 
     if len(time1) == 0 or len(time2) == 0:
         logging.info('No coinc triggers in one, or both, ifos.')
         return numpy.array([])
 
-    indices = []
     if numpy.isfinite(slide):
-        time = (time2 + (time1 + timeslide_id * slide)) / 2
+        time = (time1 + (time2 + timeslide_id * slide)) / 2
```

Comment:
unclear why this change has been made. if anything, would be clearer to just remove the interior brackets .. it's just 3 things added to each other

Reply:
I had intended it to show that it is time2 which is being shifted, however I have removed the brackets